serving/gke-assigning-static-ip-address.md

```diff
@@ -5,13 +5,12 @@ If you are running Knative on Google Kubernetes Engine and want to use a
 configure a static IP address to ensure that your custom domain mapping doesn't
 break.
 
-Knative uses the shared `knative-shared-gateway` Gateway under the
-`knative-serving` namespace to serve all incoming traffic within the Knative
+Knative configures an Istio Gateway CRD named `knative-ingress-gateway` under
```

Comment:
Yes, this is a little bit confusing because the `istio-ingressgateway` is the Istio service that generate the IP address, while `knative-ingress-gateway` is the Gateway CRD to tell our networking layer to use `istio-ingressgateway`.

I can see that the name `knative-ingress-gateway` (of the Gateway CRD) adds to the confusion, but that is already named in the manifest so it is a little bit tricky to change at this point.

Reply:
Ah, okay. Good to know. I'll see if we can find a good spot to document the distinction.